config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/EncassEntityBuilder.java

```diff
@@ -72,7 +73,13 @@ private Entity buildEncassEntity(Bundle bundle, String name, Encass encass, Docu
                 buildResults(encass, document)
         );
 
-        buildAndAppendPropertiesElement(ImmutableMap.of(PALETTE_FOLDER, INTERNAL_ASSERTIONS), document, encassAssertionElement);
+        Map<String, Object> properties = encass.getProperties();
```

Comment:
do something here using the `Optional` class:
`Optional.ofNullable(encass.getProperties()).orElse(new HashMap)`

Then you don't need the if/else below